test/wdio/shadow-dom-slot-nested/cmp.test.tsx

```diff
@@ -1,19 +1,23 @@
-import { setupDomTests } from '../util';
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
 
 describe('shadow-dom-slot-nested', () => {
-  const { setupDom, tearDownDom } = setupDomTests(document);
-  let app: HTMLElement;
-
   beforeEach(async () => {
-    app = await setupDom('/shadow-dom-slot-nested/index.html');
+    render({
+      template: () => (
+        <main>
+          main content
+          <shadow-dom-slot-nested-root></shadow-dom-slot-nested-root>
+        </main>
+      ),
+    });
   });
-  afterEach(tearDownDom);
 
   it('renders children', async () => {
-    let elm = app.querySelector('main');
+    let elm = document.querySelector('main');
     expect(window.getComputedStyle(elm).color).toBe('rgb(0, 0, 255)');
```

Comment:
Wouldn't we need [these styles](https://github.com/ionic-team/stencil/pull/5554/files#diff-5167e5130b3d8467e0db353baf3f708a63407ec2e309d1aec56d58b17f675807L7-L10) applied at the root in order for this assertion to pass? Or is there somewhere else these are set that I'm not seeing?